server/router_utils.go

```diff
@@ -147,6 +153,44 @@ func readOrgID(writer http.ResponseWriter, request *http.Request) (types.OrgID,
 	return types.OrgID(orgID), true
 }
 
+// readClusterList retrieves list of clusters from request's path
+// if it's not possible, it writes http error to the writer and returns false
+func readClusterListFromPath(writer http.ResponseWriter, request *http.Request) ([]string, bool) {
+	rawClusterList, err := getRouterParam(request, "cluster_list")
+	if err != nil {
+		handleServerError(writer, err)
+		return []string{}, false
+	}
+
+	// basic check that should not happen in reality (because of Gorilla mux checks)
+	if rawClusterList == "" {
+		handleServerError(writer, errors.New("cluster list is empty"))
+		return []string{}, false
+	}
+
+	// split the list into items
+	clusterList := strings.Split(rawClusterList, ",")
+
+	// everything seems ok -> return list of clusters
+	return clusterList, true
+}
+
+// readClusterList retrieves list of clusters from request's body
```

Comment:
there are two comments, here and at line that are the same, is this correct?

Reply:
good catch, fixed!